### PR TITLE
Fixed issue missing parent class name when localize in nested class

### PR DESCRIPTION
**Pull request by dung691**
Fixed [#68](https://github.com/OrchardCoreContrib/OrchardCoreContrib.PoExtractor/issues/68)

### CODE REVIEW

File: src/OrchardCoreContrib.PoExtractor.DotNet.CS/MetadataProviders/CSharpMetadataProvider.cs [hunk 1]
```diff
@@ -29,8 +30,14 @@ public string GetContext(SyntaxNode node)
                 .Name.ToString();
             var @class = node.Ancestors()
                 .OfType<ClassDeclarationSyntax>()
-                .FirstOrDefault()?
-                .Identifier.ValueText;
+                .Select(x => x.Identifier.ValueText)
```
Comment (hishamco) [inline src/OrchardCoreContrib.PoExtractor.DotNet.CS/MetadataProviders/CSharpMetadataProvider.cs:33]: Just few comments:

- We should do this for both C# & VB
- Adding a unit test will be a plus

Comment (dung691) [reply src/OrchardCoreContrib.PoExtractor.DotNet.CS/MetadataProviders/CSharpMetadataProvider.cs:33]: sorry, i am not family with VB. feel free to close this PR and add another

**Comment by hishamco** [comment]
In case you need a help on unit testing, just let me know I can handle it

A side note: we could merge this after `1.0.0` to avoid unexpected results

**Comment by hishamco** [comment]
@dung691 I think it's a time to look into this one, could you please add a unit test to your PR. Also it will valuable if you provide an example for the mentioned bug, so we can sure this PR fix it

**Comment by dung691** [comment]
@hishamco i updated issue with attached project. Please take a look,

Sorry, i have no idea how to unit test this

**Comment by hishamco** [comment]
I might added a unit tests to ensure your fix it doing well

**Comment by hishamco** [comment]
Closed in favor of #74

**Pull request closed without merging**